mldsa/polyvec.c

```diff
@@ -108,10 +108,24 @@ void polyvecl_uniform_gamma1(polyvecl *v, const uint8_t seed[MLDSA_CRHBYTES],
                              uint16_t nonce)
 {
   unsigned int i;
+  uint16_t n = MLDSA_L * nonce;
```

Comment:
What's the benefit of introducing this? Can we not stick to `MLDSA_L * nonce + i` in the loop?